lnrpc/walletrpc/walletkit_server_test.go

```diff
@@ -0,0 +1,62 @@
+//go:build walletrpc
+// +build walletrpc
+
+package walletrpc
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/lightningnetwork/lnd/input"
+	"github.com/stretchr/testify/require"
+)
+
+// TestWitnessTypeMapping tests that the two witness type enums in the `input`
+// package and the `walletrpc` package remain equal.
+func TestWitnessTypeMapping(t *testing.T) {
+	t.Parallel()
+
+	// tests that both enum types have the same length except the
+	// UNKNOWN_WITNESS type which is only present in the walletrpc
+	// witness type enum.
+	require.Equal(
+		t,
+		len(allWitnessTypes),
+		len(WitnessType_name)-1,
+		"number of witness types should match proto definition")
+
+	// tests that both enum types map to the same numerical value and their
+	// string representations are equivalent.
+	for witnessType, witnessTypeProto := range allWitnessTypes {
+		// redeclare to avoid loop variables being captured
+		// by func literal.
+		witnessType := witnessType
+		witnessTypeProto := witnessTypeProto
+
+		//nolint:lll
```

Comment:
`require.EqualValues` would have been better. In any case, since it no longer applies, I have dropped this part of the test, and also dropped the`nolint:lll`.